prototypes.js

```diff
@@ -0,0 +1,53 @@
+// Choose a topic/theme that you’re interested in.
+// For example Movies, Cars, Music, Sports etc.
+// (Don’t use the examples from the slides)
+
+// Implement the below prototypal inheritance diagram using the topic you chose
+// Create some instances with the child constructor functions and call the prototype methods
+
+function GenshinImpactCharacter(weapon, vision, rarity) {
+    this.weapon = weapon;
+    this.vision = vision;
+    this.rarity = rarity;
+    this.getConstellationNumber = function () {
```

Comment:
it would be better to set these methods on the prototype of GenshinImpactCharacter.
GenshinImpactCharacter.prototype.getConstellationNumber = function() {}.
In this way all your instance of subclass will get a link to these methods, but not have a personal copy